components/feature/app-links/src/test/java/mozilla/components/feature/app/links/AppLinksInterceptorTest.kt

```diff
@@ -154,6 +154,12 @@ class AppLinksInterceptorTest {
         assertEquals(null, response)
     }
 
+    @Test
+    fun `request is not intercepted if not user gesture but is a subframe request`() {
```

Comment:
nit: can we reword this to "request is not intercepted if a subframe request and not triggered by user"?

Reply:
Sure, I'll make the change